### DISCUSSION TITLE
Measuring header height in Android

### DISCUSSION THREAD

**Discussion by ha3** [software-mansion/react-native-screens#756] [Q&A] [answered]
Hello,

I am trying to implement a custom search bar header to my screen. I want it to have the same height as a native header. Readme explains how to measure header height in iOS but I couldn't find anything about Android. Is there a way to get header height in Android?

Thank you,

**Comment by WoLewicki** [comment, marked as answer]
@ha3 can you check if https://github.com/software-mansion/react-native-screens/pull/1031 is what you need?

**Comment by tboba** [comment]
Hi! Just wanted to add some words there - react-native-screens 3.26.0 introduces new `useAnimatedHeaderHeight` and `useReanimatedHeaderHeight` hooks which lets you calculate header's height **dynamically**. If it's your case here, I would suggest using these hooks, but be careful of using them in heavy views/components, as they may decrease your app performance.

Cheers 🤞
